src/main/java/racingcar/util/CarNameValidator.java

```diff
@@ -0,0 +1,46 @@
+package racingcar.util;
+
+public class CarNameValidator {
+
+    public static final String WHITE_SPACE_REGEX = "\\s";
+    public static final String VALID_CHARACTER_REGEX = "^[a-zA-Z가-힣0-9]*$";
+
+    public static void validate(String carName) {
+
+        isEmpty(carName);
+        validateRange(carName);
+        hasSymbol(carName);
+        hasWhiteSpace(carName);
+    }
```

Comment:
validator로 검증을 책임 분리하는 방법은 좋은 거 같습니다! 다만, void보다 validator의  인자값을 리턴하게 하면 테스트할때도 유용할거라 생각합니다!

```suggestion
    public static String validate(String carName) {

        isEmpty(carName);
        validateRange(carName);
        hasSymbol(carName);
        hasWhiteSpace(carName);
        
        return carName
    }
```

Reply:
저는 검증 로직이 검증만 하고 변환 관련 책임이 없는 지금이 바람직해 보입니다!!

Reply:
네 생각해보니 꼭 반환값을 넘길 필요는 없다 생각하네요 고려할 필요는 없을 거 같습니다!